music/admin.py

```diff
@@ -0,0 +1,7 @@
+from django.contrib import admin
+
+from .models import Album
+from .models import Song
+
+admin.site.register(Album)

```

Comment:
I suggest to use this method for Admin models:
```
@admin.register(Album)
class AlbumAdmin(admin.ModelAdmin):
      pass

```

Cause this allows you to customize later